lib/cmake_install.cmake

```diff
@@ -0,0 +1,57 @@
+# Install script for directory: /u/magina/Documents/lehrfempp
```

Comment:
This one has also been created by CMake, please remove it!

Reply:
Removed.